openapi/paylocity/client.bal

```diff
@@ -0,0 +1,442 @@
+import  ballerina/http;
```

Comment:
License header missing

Reply:
Fixed in https://github.com/ballerina-platform/ballerinax-openapi-connectors/commit/30ae73f063391071311ed2f7f5a9639bfe05a36c